components/gitpod-protocol/src/workspace-cluster.ts

```diff
@@ -104,6 +104,7 @@ export interface WorkspaceClusterDB {
      */
```

Comment:
WE should definitely adjust `deleteBy` and `findBy` as well, and make them take a composite of the shape `Pick<WorkspaceCluster, "name" | "applicationCluster">`

Reply:
That's done in c7c3e00fc and 0abf8ea15.